spec/lucky/action_rendering_spec.cr

```diff
@@ -81,7 +81,7 @@ end
 
 class Rendering::Text::WithTypedStatus < Lucky::Action
   get "/foo" do
-    text "Anything", status: Status::Created
+    text "Anything", status: HTTP::Status::CREATED
```

Comment:
Could you switch these all to symbols? I think that is the way we should use these from now on both in the source and in the guides